tests/test_0001_html.py

```diff
@@ -258,3 +258,86 @@ def test_attribute_list():
     d.id_list.toggle('foo')
 
     assert d.id_list == ['foo', 'bar']
+
+
+@pytest.mark.dependency()
```

Comment:
nit: Do we need add this line to each test?

Reply:
@maratori Currently yes. I am not very happy with how this works right now, but the idea was that you can have low-level tests (pretty much all tests at the moment are low-level tests) and high-level tests that involve a browser etc, and high-level tests don't run if the low-level tests fail.